skeleton/README.md

```diff
@@ -0,0 +1,152 @@
+# IOHK Skeleton Project
```

Comment:
I suggest listing all the files and directories that are in here with a sentence, e.g. the `.gitignore`, the `.gitattributes`, the `.buildkite` folder.

Reply:
I agree, there are quite some files and directories, but a brief description of the most important ones could help:

![image](https://user-images.githubusercontent.com/175315/61706149-a724c980-ad47-11e9-9faa-3f453b5b832c.png)